src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -129,8 +129,7 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
             (IssuanceToken, address, BondingCurveProperties, address)
         );
 
-        ERC20Issuance_v1 _issuanceToken = new ERC20Issuance_v1();
-        _issuanceToken.init(
+        ERC20Issuance_v1 _issuanceToken = new ERC20Issuance_v1(
             string(abi.encodePacked(issuanceTokenData.name)),
             string(abi.encodePacked(issuanceTokenData.symbol)),
```

Comment:
Why is this being converted from `string` (original type according to the `IssuanceToken` struct) to `string` again via `encodePacked`? Am I missing something, or is this just not doing anything?

Reply:
Not sure tbh. I removed it

Reply:
That's probably on me. We were using bytes32 before (fabi brought it up in the sc-chat) and I changed it to string at some point. Must have missed that one when refactoring